pkg/sources/reconciler/googlecloudstoragesource/pubsub.go

```diff
@@ -155,7 +155,7 @@ func ensurePubSubTopic(ctx context.Context, cli *pubsub.Client) (*v1alpha1.GClou
 			Labels: pubsubResourceLabels(src),
 		}
 
-		topic, err = cli.CreateTopicWithConfig(ctx, topicID, cfg)
+		topicCreated, err := cli.CreateTopicWithConfig(ctx, topicID, cfg)
```

Comment:
(If you prefer to use the `topicCreated` variable and apply that change to the aforementioned sources that's also fine with me.)

Reply:
thanks for your review and feedback! i will apply the similar changes to the other files, I also was looking maybe to refactor this code, but this can be done later on

Reply:
Sure, as long as the changes preserve the feedback via API events and status changes (especially on errors) feel free to send a follow-up!

Reply:
@antoineco went in another approach, removing the `topicCreated` because we only use that to print the topic name, so using the `topicID` as well. If you prefer to keep that variable, let me know I can revert

I'm taking a better look and see to remove all the duplication code, if you agree i can work on that and open a PR for discussion.

again thanks for your review and feedback

Reply:
Printing `topic` or `topicID` should be more or less equivalent.

`topic` is returned in the format `project/<project>/topic/<id>` whereas `topicID` holds only the last segment. IMO that's absolutely fine.